physionet-django/project/templates/project/published_project.html

```diff
@@ -331,10 +331,17 @@ <h5>Access the files</h5>
             </li>
           {% endif %}
           {% if has_s3_credentials and project.aws.sent_files and s3_uri %}
+          {% if project.access_policy == AccessPolicy.OPEN or user.cloud_information and user.cloud_information.aws_verification_datetime %}
```

Comment:
@Chrystinne please could you give a little more detail on the changes here? e.g. 

- why drop `has_s3_credentials` and `s3_uri`? Are these incorrect or not needed?
- what is `user.cloud_information` checking? does this return true if GCP details are provided but not AWS?
- if `project.access_policy == AccessPolicy.OPEN` is true, it will always display the AWS command won't it? What happens if the data isn't on AWS?